cmd_utils.py

```diff
@@ -44,6 +44,7 @@ def add_base_arguments(parser, default_help):
     mono_sources_default = os.environ.get('MONO_SOURCE_ROOT', '')
 
     parser.add_argument('--verbose-make', action='store_true', default=False, help=default_help)
+    parser.add_argument('--jobs', default='1', help=default_help)
```

Comment:
Could likely be `int` instead of `str` but I'm not familiar with ArgumentParser and I just went for the simplest solution. That means no validation, but if users pass garbage, `make` will complain.

Reply:
I think this is done by passing `type=int`, but I'm not too familiar with it either.

Reply:
Yeah I tried that first but then got errors since `int` is not iteratable... I figured `str` works just fine for this use case.